mspm0/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+add_subdirectory(source/ti/devices)
```

Comment:
Need a README under mspm0 directory detailing how these files were generated and how someone can update to a newer SDK , also please add details on what upstream SDK version these sources were generated, see
https://github.com/zephyrproject-rtos/hal_ti/blob/master/simplelink/README

Reply:
+1